### PR TITLE
Data inspection notebook and setup shell script

### PR DESCRIPTION
**Pull request by shubhamagarwal92**
1. Data inspection notebook 
2. Shell script for setting up on linux (Tested on ubuntu)

### CODE REVIEW

File: setup_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Following the structue:
```
Comment (Rebecca-Qian) [inline setup_dependencies.sh:3]: Nit: typo

File: setup_dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# -minecraft
+#    -- code
+#    -- data
+#    -- models
+
+mkdir -p ../data
+cd ../data
+
+wget https://craftassist.s3-us-west-2.amazonaws.com/pubr/dialogue_data.tar.gz
```
Comment (Rebecca-Qian) [inline setup_dependencies.sh:12]: you can also do `wget -P <dir_path>`

File: setup_dependencies.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+wget https://craftassist.s3-us-west-2.amazonaws.com/pubr/house_data.tar.gz
+
+tar -zxvf dialogue_data.tar.gz
+tar -zxvf instance_segmentation_data.tar.gz
+tar -zxvf house_data.tar.gz
+
+## For Ubuntu
+# Glog
+# https://codeyarns.com/2017/10/26/how-to-install-and-use-glog/
+sudo apt install libgoogle-glog-dev
```
Comment (snyxan) [inline setup_dependencies.sh:23]: Is there a way to NOT require sudo permission in this script?

Comment (shubhamagarwal92) [reply setup_dependencies.sh:23]: I can remove sudo from `apt-get install`. Any suggestion otherwise?

File: setup_dependencies.sh (hunk 2 above)
Comment (shubhamagarwal92) [reply setup_dependencies.sh]: please check

**Comment by shubhamagarwal92** [comment]
@Rebecca-Qian @snyxan can you also resolve the merge conflict in .gitignore please and allow .ipynb in the repo?

**Comment by snyxan** [comment]
Hi @shubhamagarwal92 ,

After discussion we came to a conclusion that the `.ipynb` file should not be put into the repository: the data inspection script is not part of the source code and should be kept locally only for developer's convenience. Sorry for not clarifying in advance. You are welcome to add some guidance to README where you see fit. Closing this issue now.

Also, any contributions to the features/functionalities of CraftAssist is appreciated. Thanks!

**Pull request closed without merging**